app/main/debianupdater.js

```diff
@@ -0,0 +1,40 @@
+const { app } = require('electron');
+const { Notification } = require('electron');
+
+const request = require('request');
+const semver = require('semver');
+const ConfigUtil = require('../renderer/js/utils/config-util');
+
+function updateDebianNotification() {
+	let url = '';
+
+	if (ConfigUtil.getConfigItem('betaUpdate')) {
+		url = 'https://api.github.com/repos/zulip/zulip-electron/releases';
+	} else {
+		url = 'https://api.github.com/repos/zulip/zulip-electron/releases/latest';
+	}
+
+	const options = {
+		url,
+		headers: {'User-Agent': 'request'}
+	};
+
+	request(options, (error, response, body) => {
+		if (error) {
+			console.log('Error:', error);
+			return;
+		}
+		if (response.statusCode < 400) {
+			const data = JSON.parse(body);
+			if (semver.gt(data.tag_name, app.getVersion())) {
```

Comment:
In case of pre-release i.e. `https://api.github.com/repos/zulip/zulip-electron/releases`, it will be `data[1].tag_name`.

Reply:
Works suitably on `data[0].tag_name` . Pushing that.